src/user-guide/writing-workflows/scheduling.rst

```diff
@@ -998,82 +944,67 @@ name in the graph) and, and off the clock, and arbitrary external events.
 External triggering is relatively more complicated, and is documented
 separately in :ref:`Section External Triggers`.
 
-
 Success Triggers
 ^^^^^^^^^^^^^^^^
 
-The default, with no trigger type specified, is to trigger off the
+The default, with no trigger type specified, is to trigger off of the
```

Comment:
As a Brit I can't say I approve of "off of", but I guess this may be down to opinion

Reply:
Discussion of this might have predated you joining the team @MetRonnie. Kiwis side with Brits on this, but unfortunately in this context our way can be ambiguous:

By *B triggers off A* we mean this:
```
A => B
```
but it can easily be read the wrong way around as *B triggers A*  (c.f. *B kicks off A*).  But *B triggers off of A* is unambiguous. 

Plus, according to the OED "off of" actually has a long history in British English, although it has fallen out of favour more recently.

And you can't argue it's grammatically wrong to have two prepositions in a row. Even Brits would *out of* all the time, no?  ("Pull your head out of your arse", etc.).

Reply:
See, I told you I was also a pedant :rofl: